src/view/template.js

```diff
@@ -89,7 +89,7 @@ Candy.View.Template = (function(self){
 
 	self.Message = {
 		pane: '<div class="message-pane-wrapper"><ul class="message-pane"></ul></div>',
-		item: '<li><small data-timestamp="{{timestamp}}">{{time}}</small><div>' +
+		item: '<li data-message-id="{{id}}"><small data-timestamp="{{timestamp}}">{{time}}</small><div>' +
```

Comment:
I meant just `id`, not `data-id` :D but then you'd probably need a `message-` prefix to avoid collisions..